src/self-managed-debug/src/system_catalog_dumper.rs

```diff
@@ -435,14 +443,6 @@ const RELATIONS: &[Relation] = &[
         name: "mz_dataflow_operator_dataflows",
         category: RelationCategory::Introspection,
     },
-    Relation {
-        name: "mz_dataflow_operator_reachability_per_worker",
-        category: RelationCategory::Introspection,
-    },
-    Relation {
-        name: "mz_dataflow_operator_reachability",
-        category: RelationCategory::Introspection,
-    },
```

Comment:
Did you mean to include this? I guess we don't want to scrape this info anymore?

Reply:
Yeah exactly haha! These queries were taking like 4 minutes each 👻